core/src/com/fallenflame/game/FogController.java

```diff
@@ -50,11 +69,15 @@ public void updateFog(Vector2 scale) {
                     fogArr.removeValue(effect, true);
                 }
                 /*Only make a new fog particle if we do not have enough particles in the array for that tile*/
-                if (fogArr.size < 2|| (levelModel.hasEnemy(x, y) && fogArr.size < 9)) {
+                if (fogArr.size < 2|| (levelModel.hasEnemy(x, y) && fogArr.size < 12)) {
                     ParticleEffectPool.PooledEffect effect = fogPool.obtain();
-                    for (int i = 0; i < (1 + (levelModel.hasEnemy(x, y) ? 8 : 1)); i++) {
-                        float randomVal = (float) (Math.random() * TILE_SIZE);
-                        effect.setPosition((x * TILE_SIZE + randomVal) * scale.x, (y * TILE_SIZE + randomVal) * scale.y);
+                    float incX = (float) ((Math.random() - 0.5) * 2) * 3;
+                    float incY = (float) ((Math.random() - 0.5) * 2) * 3;
+                    for (int i = 0; i < (1 + (levelModel.hasEnemy(x, y) ? 11 : 1)); i++) {
+                        float randomVal = levelModel.hasEnemy(x,y) ? 2.0f : 1.0f;
+                        float randomX = (float) (((Math.random()*randomVal)-0.5f) * TILE_SIZE);
+                        float randomY = (float) (((Math.random()*randomVal)-0.5f) * TILE_SIZE);
+                        effect.setPosition(((x+incX) * TILE_SIZE + randomX) * scale.x, ((y+incY) * TILE_SIZE + randomY) * scale.y);
```

Comment:
Can we factor out randomization constants here like the 0.5, 2, and 3 from `((Math.random() - 0.5) * 2) * 3` and have them defined as constants at the top of the class with descriptions on what their values change? That way we can more easily tweak the randomization to our likes. Also why is it *2 then *3? Why not *6?

Reply:
Lmao what happens when tired coders try to do math. Resolved in new push